tests/bodyxml-to-content-tree/input/simple-old-post.xml

```diff
@@ -8,5 +8,5 @@
     <pull-quote-text><p>Maecenas ac ipsum in elit aliquam consectetur. Proin felis metus, efficitur et nulla eu, interdum malesuada diam.</p></pull-quote-text><pull-quote-image><ft-content data-embedded="true" url="http://api-t.ft.com/content/77c8a5b5-c9e3-4df2-ad5f-3ef35fe1d9d4" type="http://www.ft.com/ontology/content/ImageSet"></ft-content></pull-quote-image><pull-quote-source>Pellentesque habitant, morbi tristique</pull-quote-source>
 </pull-quote>
 <p>Donec id faucibus erat. Suspendisse tempor laoreet lorem, sit amet vehicula massa facilisis at. Nulla quis feugiat massa. Praesent viverra non lectus ut ullamcorper. Phasellus <ft-content url="http://api-t.ft.com/content/c71efed9-fe5a-488d-9f47-20c15d177153" type="http://www.ft.com/ontology/content/Article">porttitor neque</ft-content> at volutpat pulvinar.</p>
-<p>“Curabitur fermentum, dolor vel interdum varius, tellus justo dapibus velit, interdum sollicitudin dolor nibh varius velit.”</p>
+<p>“Curabitur fermentum, dolor vel interdum varius, tellus justo dapibus velit, interdum sollicitudin dolor nibh varius velit.”</p> 
```

Comment:
Added a space to the input XML, so that our tests cover it (there should be no change in the output)